resallocserver/maint.py

```diff
@@ -59,6 +60,19 @@ def resource_delete(self, resources=None):
                 resources = QResources(session=session)
                 resources.kill(res_id)
 
+    def resource_logs(self, resources=None):
```

Comment:
Wish ... it would be awesome if this accepted both resource name and resource ID, somehow.

Reply:
So that somebody could do either

    resource-maint resource-logs 123

or

    resource-maint resource-logs copr_hv_x86_64_02_dev_00021190_20220108_121757

?

Reply:
I think that one should by default (either id or name) and we should have an option for the other variant.

Reply:
Do you think an option is necessary? It seems to be unnecessary to me. PTAL at the newest commit.